tests/storage/gridfs_media_storage_test.py

```diff
@@ -0,0 +1,53 @@
+
+import io
+import bson
+import flask
+import unittest
+from unittest.mock import Mock
+from superdesk.upload import bp
+from superdesk.datalayer import SuperdeskDataLayer
+from superdesk.storage.desk_media_storage import SuperdeskGridFSMediaStorage
+
+
+class GridFSMediaStorageTestCase(unittest.TestCase):
+
+    def setUp(self):
+        self.app = flask.Flask(__name__)
+        self.app.config['SERVER_NAME'] = 'localhost'
+        self.app.config['DOMAIN'] = {'upload': {}}
+        self.app.data = SuperdeskDataLayer(self.app)
+        self.media = SuperdeskGridFSMediaStorage(self.app)
+        self.app.register_blueprint(bp)
+
+    def test_media_id(self):
+        filename = 'some-file'
+        media_id = self.media.media_id(filename)
+        self.assertIsInstance(media_id, bson.ObjectId)
+        self.assertEqual(media_id, self.media.media_id(filename))
```

Comment:
You should supply a calculated value here instead of the result of the call to `self.media.media_id(filename)`